BackEnd/scripts/generateTeamModel.ts

```diff
@@ -29,16 +29,27 @@ type RisenSheetRow = {
 
 async function buildTeam(seasonId: number, displayName: string, abbreviation: string, win: number, loss: number, opgg: string) {
   await ensureConnection();
+  let team = await findTeam(displayName, abbreviation, seasonId);
+
   if(await findTeam(displayName, abbreviation, seasonId)) {
```

Comment:
Duplicate call here. Just replace with `if(team)`

Reply:
Oh lmao, i fixed this in the actual code but not the script, will fix